ra/ra.go

```diff
@@ -1837,7 +1837,9 @@ func (ra *RegistrationAuthorityImpl) recordValidation(ctx context.Context, authI
 // countFailedValidation increments the failed authorizations per domain per
 // account rate limit. There is no reason to surface errors from this function
 // to the Subscriber, spends against this limit are best effort.
-func (ra *RegistrationAuthorityImpl) countFailedValidation(ctx context.Context, regId int64, name string) {
+func (ra *RegistrationAuthorityImpl) countFailedValidation(ctx context.Context, regId int64, iden identifier.ACMEIdentifier) {
```

Comment:
nit: we [usually say](https://github.com/letsencrypt/boulder/blob/844334e04aef79d4fabc08aab9b8a95db10611ec/va/http.go#L642) "ident", not "iden"